### PR TITLE
Added the header field "X-Ably-Lib"

### PR DESCRIPTION
**Pull request by lucky-dev**
Quick fix http://docs.ably.io/client-lib-development-guide/features/#RSC7b

### CODE REVIEW

File: Spec/RestClient.swift [hunk 1]
```diff
@@ -29,6 +29,8 @@ class RestClient: QuickSpec {
                         expect(error).to(beNil())
                         let version = testHTTPExecutor.requests.first!.allHTTPHeaderFields?["X-Ably-Version"]
                         expect(version).to(equal("0.8"))
+                        let libVersion = testHTTPExecutor.requests.first!.allHTTPHeaderFields?["X-Ably-Lib"]
+                        expect(libVersion).to(equal("0.8.3"))
```
Comment (mattheworiordan) [inline Spec/RestClient.swift]: How does this pass given libVersion is supposed to be `ios-0.8.3`?

Comment (lucky-dev) [inline Spec/RestClient.swift]: Fixed.

File: Spec/RestClient.swift [hunk 2]
```diff
@@ -154,10 +154,10 @@ class RestClient: QuickSpec {
                     let client = ARTRest(options: options)
 
                     client.logger.log("This is a warning", withLevel: .Warn)
-                    
+
```
Comment (mattheworiordan) [inline Spec/RestClient.swift]: Why did you commit all these white space changes?

Comment (lucky-dev) [inline Spec/RestClient.swift]: It's very strange. I didn't do any extra white spaces.

Comment (mattheworiordan) [inline Spec/RestClient.swift]: Your editor made those changes, but you then committed it.

File: Spec/RestClient.swift [hunk 3]
```diff
@@ -30,7 +30,7 @@ class RestClient: QuickSpec {
                         let version = testHTTPExecutor.requests.first!.allHTTPHeaderFields?["X-Ably-Version"]
                         expect(version).to(equal("0.8"))
                         let libVersion = testHTTPExecutor.requests.first!.allHTTPHeaderFields?["X-Ably-Lib"]
-                        expect(libVersion).to(equal("0.8.3"))
+                        expect(libVersion).to(equal("ios-0.8.3"))
```
Comment (mattheworiordan) [inline Spec/RestClient.swift]: Did you not run the test before?  Does this test pass?

Comment (lucky-dev) [inline Spec/RestClient.swift]:
```
Test Suite 'RestClient' started at 2016-08-02 22:47:28.973
Test Case '-[AblySpec.RestClient RestClient__All_REST_requests_should_include_the_current_API_version]' started.
Test Case '-[AblySpec.RestClient RestClient__All_REST_requests_should_include_the_current_API_version]' passed (7.302 seconds).
```

**Pull request closed without merging**